src/docs/get-started/install/_ios-setup.md

```diff
@@ -71,41 +71,17 @@ To create your first Flutter app and test your setup, follow these steps:
 
 ### Deploy to iOS devices
 
-To deploy your Flutter app to a physical iOS device,
-you’ll need some additional tools and an Apple account.
+To deploy your Flutter app to a physical iOS device you will need the
```

Comment:
Please delete the "will".

Reply:
Done

Reply:
> LGTM with one minor tweak. We don't generally make a note of changes to the installation steps. (I can't remember why, but I recently changed the installation steps and tried including a note, but was asked to remove it.) I guess it could confuse some newbies. As for when stable happens, is this for 1.9 or the following release? That might be a question for either @tvolkert or @timsneath.

This is for the 1.9.1 stable release, which just went live, so I will land this now, thanks!